README.md

```diff
@@ -1,2 +1,81 @@
-# handbook-of-best-practices
-Software Quality Assurance Assignment
+# Handbook of Best Practices
+
+## Task Estimation
+
+
+The Sprint Planning meeting is the Scrum ceremony where the team estimates all tasks associated with each user story in the Sprint backlog. 
+The estimation effort focuses on all resources required to complete the tasks for each Sprint. 
+
+A shared task list is used to estimate the 
+duration and effort for each user story in the backlog. This practice results in a shared team perspective, resulting in a task estimation 
+process that is consistent and reliable throughout all Sprints. 
+In this section of the handbook we will discuss the following points : 
+
+
+ + Building a successful story point estimation
+ + The importence of story point estimation
+ + Challenges of accurately estimating tasks
+ + Overcoming estimation challenges
+ + Bad practices of task estimation
+ + Advantages and disadvantages of twice estimation
+ 
+
+&nbsp;
+
+
+### 1. Building a successful story point estimation
+
+---
+In Task Estimation, team does a comparative analysis between all of the stories for the project. For each story to be sized, 
+ the following as a team(Product Owner, Core Scrum team including developers, testers & scrum master):
+
+
+
+&nbsp;
+
+<img src="https://github.com/xtreger/handbook-of-best-practices/blob/task-estimation-in-scrum-jouhara/Images/steps.png" width="500" height="200"/>
+
+&nbsp;
+
+### 2. The importence of story point estimation
+
+---
+
+Often happens that one person estimates a project, but other people complete the tasks. In this case, Story Points are indispensable. An example of why story points better than hours is illustrated in the example below:
+
+> A company has a project at 200 story points(SP). One engineer starts working on it. His velocity is 1 SP per 3 hours. The start date is May 30th. Based on this information.
+
+it’s easy to calculate the project release date as follows:
+
+1. The time needed to finish all the work :
+
+
+    **T=Qsp*t**
+       
+    200x3 = 600 (hours).
+
+2. The number of workdays:
+
+    **Nw=T/Prt**
+       
+    600 / 6 = 100 (workdays).
+
+**Solution:** Having a calendar at hand, we can then calculate a release date — October 14th.    
+
+&nbsp;    
+
+| Formula keywords |  |
+| ------ | ----------- |
+| Qsp   | general quantity of SP |
+| T | Time |
+| Nw    | Number of days |
+| Prt    | quantity of productive hours (usually 6 hours) |
+
+
+
+&nbsp;
+
+|References |
+|---|
+|[Building a successful story point estimation](http://www.agilebuddha.com/agile/agile-estimation-8-steps-to-successful-story-point-estimation/)|
+[The importence of story point estimation](https://rubygarage.org/blog/3-reasons-to-estimate-with-story-points)|
```

Comment:
One more small mistake here that Diya mentioned earlier.

Reply:
Fixed